### PR TITLE
Feat #19 added authorization

### PR DESCRIPTION
**Pull request by nickonos**

### CODE REVIEW

**Review by WouterPennings — DISMISSED** [review]

File: src/main.ts [hunk 1]
```diff
@@ -1,6 +1,8 @@
-import { createApp } from 'vue'
 import App from './App.vue'
 import './registerServiceWorker'
 import router from './router'
+import {createApp} from "vue"
+import * as dotenv from "dotenv"
+dotenv.config()
```
Comment (NealGeilen) [inline src/main.ts:6]: This can be removed if not used

File: src/views/Authorisation.vue [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+      if (clientid == null){
+          throw new Error("client id is null or undefined")
+      }
+      window.location.href ="https://github.com/login/oauth/authorize?client_id=" + clientid
+    }
+  }
+
+})
+
+export default class Authorisation extends Vue{
```
Comment (NealGeilen) [inline src/views/Authorisation.vue:33]: Remove class if not used

File: src/views/AuthorisationRedirect.vue [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+      type: String,
+      required: true,
+      default: null
+    }
+  },
+  Status: ""
+
+})
+
+export default class AuthorisationRedirect extends Vue{
```
Comment (NealGeilen) [inline src/views/AuthorisationRedirect.vue:29]: For now, this is good, In the future please use a service class in a separate file

**Review by NealGeilen — APPROVED** [review]
Please read the changes for future uses, for now it is good

**Review by WouterPennings — APPROVED** [review]

**Pull request merged**